src/main/java/org/zalando/nakadi/exceptions/runtime/InvalidCursorException.java

```diff
@@ -58,7 +58,8 @@ public String getMessage() {
             case PARTITION_NOT_FOUND:
                 return "non existing partition " + getPartition();
             case UNAVAILABLE:
-                return "offset " + getOffset() + " for partition " + getPartition() + " is unavailable";
+                return "offset " + getOffset() + " for partition " + getPartition() + " is unavailable. " +
+                        "Patch partition offset for subscription with available offset.";
```

Comment:
I am not sure about call to action, because it is quite extraordinary situation, although, I understand your intention to reduce the operational load. could we rephrase it to something like: 
`offset 1 for partition 0 is unavailable due to data went over the retention time window"`
feel free to make wording better, but the idea is that offset is not available for some reason, and reason is not clear in the message.